x-pack/plugins/upgrade_assistant/server/lib/reindexing/reindex_actions.test.ts

```diff
@@ -22,7 +22,7 @@ import { LOCK_WINDOW, ReindexActions, reindexActionsFactory } from './reindex_ac
 
 describe('ReindexActions', () => {
   let client: jest.Mocked<any>;
-  let callCluster: jest.Mock<CallCluster>;
+  let callCluster: jest.Mock;
```

Comment:
had to delete declaration as mock doesn't implement full `CallCluster` contract

Reply:
Sounds good, I'll fix this up to be complete after this is merged (I wrote this).